public/media_engagement/ahadi_media_engagement.html

```diff
@@ -17,21 +22,40 @@
     <script type="text/javascript" src="firebase.js"></script>
 </head>
 <body>
-    <nav class="nav">
-        <div class="nav-wrapper container">
-            <img class='brand-logo' src="avfdashboardlogo.png">
-            <ul class="right">
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/coda/coding_progress.html">MESSAGE CODING</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/ahadi_media_engagement.html">AHADI_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/icraf_media_engagement.html">ICRAF_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/imaqal_media_engagement.html">IMAQAL_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/ocha_media_engagement.html">OCHA_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/undp_rco_media_engagement.html">UNDP_RCO_E&M</a></li>
-                <li><a class="nav-link" onClick="logout();">Logout</a></li>
-            </ul>
+    <!-- NAVBAR WITH DROPDOWN & RESPONSIVE TOGGLE-->
+    <nav class="navbar navbar-expand-sm navbar-brown bg-white mb-3">
+        <div class="container">
+            <a class="navbar-brand" href="#"><img class="img-fluid logo" src="avfdashboardlogo.PNG" alt="avf-logo"></a>
```

Comment:
'img-fluid' apparently has the affect of scaling the image along the X axis only, which is pretty ugly as the screen gets small. Probably better to switch to just the AVF icon when the screen gets narrow instead.

However, this is a small cosmetic thing to feel free to merge as-is and open an issue for it instead.